apps/web/pages/settings/admin/orgMigrations/removeUserFromOrg.tsx

```diff
@@ -0,0 +1,121 @@
+import type { GetServerSidePropsContext } from "next";
+import { getSession } from "next-auth/react";
+import type { TFunction } from "next-i18next";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { useState } from "react";
+import z from "zod";
+
+import { UserPermissionRole } from "@calcom/prisma/enums";
+import { getStringAsNumberRequiredSchema } from "@calcom/prisma/zod-utils";
+import { Button, TextField, Meta, showToast } from "@calcom/ui";
+
+import PageWrapper from "@components/PageWrapper";
+
+import { getLayout } from "./_OrgMigrationLayout";
+
+function Wrapper({ children }: { children: React.ReactNode }) {
+  return (
+    <div>
+      <Meta title="Organization Migration: Move a team" description="Migrates a team to an organization" />
+      {children}
+    </div>
+  );
+}
+
+const enum State {
+  IDLE,
+  LOADING,
+  SUCCESS,
+  ERROR,
+}
+
+export const getFormSchema = (t: TFunction) =>
+  z.object({
+    userId: z.union([getStringAsNumberRequiredSchema(t), z.number()]),
+    targetOrgId: z.union([getStringAsNumberRequiredSchema(t), z.number()]),
+  });
+
+export default function RemoveUserFromOrg() {
+  const [state, setState] = useState(State.IDLE);
+
+  return (
+    <Wrapper>
+      {/*  Due to some reason auth from website doesn't work if /api endpoint is used. Spent a lot of time and in the end went with submitting data to the same page, because you can't do POST request to a page in Next.js, doing a GET request  */}
+      <form
```

Comment:
Any reason we are not using `<Form>` here -> would handle values automatically without having to use e.currentTarget to get formdata.

Reply:
I moved this code from website where I was facing some weird issues long time back. Don't remember the exact reason now 😅 Moved all components to useForm now.